spec/dashboard-below-map-view.spec.js

```diff
@@ -56,15 +60,33 @@ describe('dashboard-below-map-view', function () {
 
   describe('when a time-series widget is added', function () {
     beforeEach(function () {
-      var timeSeriesWidgetModelFake = createFakeTimeSeriesWidgetModel();
-      timeSeriesWidgetModelFake.dataviewModel = createFakeDataviewModel();
-      timeSeriesWidgetModelFake.dataviewModel.layer = createFakeDataviewLayer();
-      this.widgetsCollection.add(timeSeriesWidgetModelFake);
+      this.timeSeriesWidgetModelFake = createFakeTimeSeriesWidgetModel();
```

Comment:
Why do we create this variable inside `this`?

Reply:
If we are not going to use it out of this context, I'd not create it.

Reply:
That's correct! This remains from a previous change